### PR TITLE
Integration test branch fix

### PR DESCRIPTION
**Pull request by iamdexterpark**
Changed github actions workflows.

### CODE REVIEW

File: internal/provider/network_resource_test.go [hunk 1]
```diff
@@ -20,13 +20,14 @@ func TestAccOrganizationsNetworkResource(t *testing.T) {
 				),
 			},
 
-			// Create and Read testing (network)
+			// Create and Read testing (network).
```
Comment (Rileyj53) [inline internal/provider/network_resource_test.go:23]: remove this period

File: internal/provider/network_resource_test.go [hunk 2]
```diff
@@ -20,13 +20,14 @@ func TestAccOrganizationsNetworkResource(t *testing.T) {
...
 			{
 				Config: testAccOrganizationsNetworkResourceConfig,
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestCheckResourceAttr("meraki_network.test", "name", "test_acc_network"),
 					resource.TestCheckResourceAttr("meraki_network.test", "timezone", "America/Los_Angeles"),
 					resource.TestCheckResourceAttr("meraki_network.test", "tags.#", "1"),
+					resource.TestCheckResourceAttr("meraki_network.test", "tags.#", "1"),
```
Comment (Rileyj53) [inline internal/provider/network_resource_test.go:30]: same as line above

**Review by Rileyj53 — COMMENTED** [review]
Squash your commits

File: internal/provider/network_resource_test.go (hunk 2 above)
Comment (iamdexterpark) [reply internal/provider/network_resource_test.go]: done!

File: internal/provider/network_resource_test.go (hunk 1 above)
Comment (iamdexterpark) [reply internal/provider/network_resource_test.go]: Done!

**Pull request merged**